### ISSUE TITLE
File, line and function informations missing in logged messages.

### ISSUE THREAD

**Issue by Udo2** [fpillet/NSLogger#155]
Hello,

I added NSLogger to the small iOS "ToDoList" App, which is part of the Xcode 6 Learning Tutorial:

#import "LoggerCommon.h"
#import "NSLogger.h"

int main(int argc, char * argv[]) {
    
//  LoggerSetOptions(NULL, kLoggerOption_LogToConsole);
    NSLog(@"Hello world !\n    FILE     = %s\n    LINE     = %d\n    FUNCTION = %s", __FILE__, __LINE__, __FUNCTION__);
    LoggerApp(1, @"Hello world !");
    
    @autoreleasepool {
        return UIApplicationMain(argc, argv, nil, NSStringFromClass([AppDelegate class]));
    }
}

Basically, it works fine (compiling, connecting to the Desktop Viewer, logging).
However, I cannot get the __FILE__, __LINE__ and __FUNCTION__ informations displayed in the logged messages/output:

22:57:38.0642      Main thread | Hello world !
    FILE     = /Users/user/Documents/SoftwareDevelopment/ToDoList NSLogger/ToDoList/main.m
    LINE     = 17
    FUNCTION = main
22:57:38.0642      Main thread | Hello world !

Any hint, comments and suggestions welcome.

Sincerly Udo

**Comment by MonsieurDart** [comment]
What were you expecting? Because FUNCTION is indeed main, LINE seems to be 17 and FILE also seems right…

**Comment by Udo2** [comment]
Hello,

I would expect that the __FILE__, __LINE__ and __FUNCTION__ information is somehow logged/displayed, when I use LoggerApp(). I cannot see (or find) this very useful information displayed anywhere in the NSLogger Desktop Viewer output.

P.S.: The first NSLog call was just to see the content, I was expecting and making sure the preprocessor works. In general I would prefer using LoggerApp, focus on the message (and level) only and let the pre-processor macro complete it with the useful information.

Sincerly Udo 

 Am 23.02.2015 um 09:59 schrieb Mathieu Godart <notifications@github.com>:
> 
> What were you expecting? Because FUNCTION is indeed main, LINE seems to be 17 and FILE also seems right…
>

**Comment by fpillet** [comment]
Udo, the logging macros that @MonsieurDart develop indeed do report the file / line / function. In the desktop logger, they do not appear directly in the message. Rather, they appear in a separately colored area at top of the message. You can toggle this display by pressing the ƒ button at bottom of the log window.
